src/moveit_custom_api.cpp

```diff
@@ -281,19 +345,37 @@ void MoveitCustomApi::addCollissionObjects()
   // Define a box to add to the world.
   primitive.dimensions[0] = TOTAL_INNER_CELL_X_DIMENSION_;
   primitive.dimensions[1] = 0;
-  primitive.dimensions[2] = TOTAL_INNER_CELL_Z_DIMENSION;
+  primitive.dimensions[2] = TOTAL_INNER_CELL_Z_DIMENSION_;
 
   // Define a pose for the box (specified relative to frame_id
   box_pose.orientation.w = 1.0;
-  box_pose.position.x = -(BASE_OFFSET_FROM_LEFT_WALL_-BASE_OFFSET_FROM_RIGHT_WALL_)/2;
+  box_pose.position.x = -(BASE_OFFSET_FROM_LEFT_WALL_ - BASE_OFFSET_FROM_RIGHT_WALL_) / 2;
   box_pose.position.y = -BASE_OFFSET_FROM_BACK_WALL_;
-  box_pose.position.z = TOTAL_INNER_CELL_Z_DIMENSION/2;
+  box_pose.position.z = TOTAL_INNER_CELL_Z_DIMENSION_ / 2;
 
   object.object.operation = object.object.ADD;
   object.object.primitives.push_back(primitive);
   object.object.primitive_poses.push_back(box_pose);
   planning_scene.world.collision_objects.push_back(object.object);
 
+  // adding base for prbt workspace
+  if (robot == "prbt")
```

Comment:
Hmm you will have to find some way to generalise this as well then. It makes sense to add bottom collission object as well for every robot cell. For UR the dimensions can be 0, for PRBT non zero. Can still be configured outside.